test/smoke/usm-locals-xnack-enabled-xnack-any-gfx94x/Makefile

```diff
@@ -0,0 +1,22 @@
+include ../../Makefile.defs
+
+TESTNAME     = usm_locals
+TESTSRC_MAIN = usm_locals.cpp
+TESTSRC_AUX  =
+TESTSRC_ALL  = $(TESTSRC_MAIN) $(TESTSRC_AUX)
+RUNENV      += HSA_XNACK=1 LIBOMPTARGET_KERNEL_TRACE=2
+
+RUNCMD       = ./$(TESTNAME) 2>&1 | $(AOMP)/bin/FileCheck $(TESTSRC_MAIN)
+
+OMP_FLAGS    = -fopenmp --offload-arch=$(AOMP_GPU)
+CFLAGS       += -O3
+CLANG        = clang++
+OMP_BIN      = $(AOMP)/bin/$(CLANG)
+CC           = $(OMP_BIN) $(VERBOSE)
+
+SUPPORTED    = gfx940,gfx941,gfx942
```

Comment:
Since the gfx90a is explicitly excluded, is it problematic that the test will run on a MI300X? Should this test run on an APU only?

Reply:
At the moment it is a problem. There is just no way to distinguish between X and A. To resolve this, the CI needs to include an additional flag which specifies if the underlying arch we are running on is an APU or not.

Reply:
I will prepare the test for that case though. For non APU gfx94* copies are expected according to the tables.